src/style.css

```
@@ -1 +1,68 @@
-/* Styles go here */
+body,
+ul {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  display: flex;
+  box-shadow: 0 4px 4px rgba(0, 0, 0, 0.25);
+  justify-content: space-between;
+  padding-right: 50px;
```

why only padding-right? it's the same from both sides
```suggestion
  padding: 0 50px;
```